hack/generate.sh

```diff
@@ -47,3 +47,12 @@ protoc --go_out=plugins=grpc:. pkg/handler-launcher-com/cmd/info/info.proto
 
 mockgen -source pkg/handler-launcher-com/notify/info/info.pb.go -package=info -destination=pkg/handler-launcher-com/notify/info/generated_mock_info.go
 mockgen -source pkg/handler-launcher-com/cmd/info/info.pb.go -package=info -destination=pkg/handler-launcher-com/cmd/info/generated_mock_info.go
+
+rm -rf cluster cluster-hack
+git clone https://github.com/davidvossel/cluster-up.git
```

Comment:
@davidvossel can we just have this in the kubevirtci repo?

Reply:
This is the wrong approach. Please use a git subrepo instead of cloning.

Reply:
> This is the wrong approach. Please use a git subrepo instead of cloning.

git submodules?  I'd prefer we avoided submodules.   https://codingkilledthecat.wordpress.com/2012/04/28/why-your-company-shouldnt-use-git-submodules/

What would be the value of submodules over this initial approach. 

I don't think that there is anything inherently wrong with the approach as it is done here. It's not possible to modify the cluster/ dir directly without causing travis to fail. Just like we "generate" copy functions, schema's, etc... this is generating the cluster-up scripts.  If you update the hash env var, then you have to commit the results just like the rest of the generated code.

Reply:
> @davidvossel can we just have this in the kubevirtci repo?

That's something we should discuss. I'm not convinced where these scripts should live quite yet. We need to discuss how to run CI against this logic externally of kubevirt/kubevirt as well.

Reply:
Imho it should be under the KubeVirt CI, in this case, update the images hashes under providers will be much easier(but in this case you will need to clone the whole kubevitrci repository)

Reply:
We can do a shallow clone `git clone --depth 1`. Yes, let's discuss this. For instance in kubernetes the clonse in prow their test-infra repo on every test run.

Reply:
@cynepco3hahue @rmohr 

I've moved cluster-up logic to kubevirtci. https://github.com/kubevirt/kubevirtci/pull/87